### PR TITLE
add function name and test

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: array-of.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ if (!Array.of) {
 				return false;
 			}
 		};
-		var of = function() {
+		var of = function of () {
```
Comment (ljharb) [inline array-of.js]: there shouldn't be a space between a function's name and its argument list

File: tests/tests.js [hunk 2]
```diff
@@ -65,3 +65,5 @@ Object.defineProperty(MyType.prototype, '0', {
 	}
 });
 assertDeepEquals(Array.of.call(MyType, 'abc'), { '0': 'abc', 'length': 1 });
+
+assertEquals(Array.of.name, 'of', 'Array#of has name "of"');
```
Comment (ljharb) [inline tests/tests.js]: this test will fail in IE, which doesn't have the name property - can you wrap it in an `if` that checks if function names are supported? My typical test is `(function foo() {}).name === 'foo'`.

(also, please add a trailing newline at the end of the file)

**Comment by JakeChampion** [comment]
@ljharb I've made the edits you suggested. I think it may be a good idea to run these tests inside the browsers themselves, what do you think?

**Comment by ljharb** [comment]
Absolutely - but the options for doing that automatically are limited. For now this LGTM

**Pull request merged**